VisaCheckout/build.gradle

```diff
@@ -31,20 +31,20 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.annotation:annotation:1.0.1'
-    implementation 'com.braintreepayments.api:braintree:3.0.0'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.braintreepayments.api:braintree-core:4.0.0-SNAPSHOT'
 
     implementation(group: 'com.visa.checkout', name: 'visacheckout-android-sdk', version: '6.6.1', ext: 'aar')
 
     testImplementation 'com.google.android.gms:play-services-wallet:16.0.1'
     testImplementation 'androidx.test:core:1.1.0'
     testImplementation 'androidx.test:runner:1.1.1'
     testImplementation 'androidx.test:rules:1.1.1'
+    testImplementation 'org.robolectric:robolectric:4.1'
```

Comment:
why did this get moved up here? Not the end of the world that it's out of a-z order

Reply:
TBH probably sheer frustration with Robolectric haha.  We can alphabetize here.